### PR TITLE
add assets and create  transformAssetData

### PR DESCRIPTION
**Pull request by lindawoo-plaid**
This PR adds assets to the quickstart when the products variable includes "assets"

image

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const App = () => {
     dispatch({
       type: "SET_STATE",
       state: {
-        paymentInitiation,
+        products: data.products,
```
Comment (skylarmb) [inline frontend/src/App.tsx:26]: nice! good improvement

File: frontend/src/dataUtilities.ts [hunk 2]
```diff
@@ -475,3 +504,28 @@ export const transformPaymentData = (data: PaymentData) => [
...
+
+interface AssetResponseData {
+  json: AssetReport;
+}
+
+export const transformAssetsData = (data: AssetResponseData) => {
+  const assetItems = data.json.items;
+  let final: DataItem[] = [];
+  assetItems.forEach((item) => {
+    final = final.concat(
```
Comment (skylarmb) [inline frontend/src/dataUtilities.ts:516]: could this be more immutable and more concise? also where is this `json` key coming from? none of the others have the data nested under `.json`

```tsx
export const transformAssetsData = (data: AssetResponseData): Array<DataItem> =>
  data.json.items.map((item) =>
    item.accounts.map((account) => ({
      account: account.name,
      balance:
        formatter.format(account.balances.available!) ||
        formatter.format(account.balances.current!),
      transactions: account.transactions!.length,
      daysAvailable: account.days_available!,
    }))
  );
```

File: python/requirements.txt [hunk 3]
```diff
@@ -6,5 +6,5 @@ click==6.7
 itsdangerous==0.24
 requests==2.20.0
 wsgiref==0.1.2 ; python_version < '3.2'
-plaid-python>=5.0.0, < 6.0.0
```
Comment (skylarmb) [inline python/requirements.txt:9]: was this intentional?

File: frontend/src/dataUtilities.ts (hunk 2 above)
Comment (lindawoo-plaid) [reply frontend/src/dataUtilities.ts]: changed to flatMap

File: python/requirements.txt (hunk 3 above)
Comment (lindawoo-plaid) [reply python/requirements.txt]: deleted

**Review by skylarmb — APPROVED** [review]

**Review by skylarmb — APPROVED** [review]

**Pull request merged**